app/views/applications/show.html.erb

```diff
@@ -1,7 +1,22 @@
-<p>Name: <%= @applications.name %>
-<br>Full Address: <%= @applications.address %>, <%= @applications.city %>, <%= @applications.state %> <%= @applications.zip %>
-<br>Description: <%= @applications.description %>
-<br>All Pets: <% @applications.pets.each do |pet| %>
+<p>Name: <%= @application.name %>
+<br>Full Address: <%= @application.address %>, <%= @application.city %>, <%= @application.state %> <%= @application.zip %>
+<br>Description: <%= @application.description %>
+<br>All Pets: <% @application.pets.each do |pet| %>
   <%= pet.name %>
 <% end %>
-<br>Application Status: <%= @applications.status %></p>
+<br>Application Status: <%= @application.status %></p>
+
+<% if @application.status == "In Progress" %>
+  <%= "Add a Pet to this Application" %>
+  <%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |form| %>
+    <br> <%= form.label :pet_name %>
+    <%= form.text_field :pet_name %>
+
+    <br> <%= form.submit "Submit" %>
+  <% end %>
+  <% if params[:pet_name] %>
```

Comment:
I think this conditional was a good solution for determining whether we should display that pet as